satpy/readers/mirs.py

```diff
@@ -356,6 +356,17 @@ def _fill_data(self, data_arr, attrs):
             data_arr = data_arr.where(data_arr != fill_value, fill_out)
         return data_arr, attrs
 
+    def _get_valid_range(self, data_arr, attrs):
+        # handle valid_range
+        valid_range = attrs.pop('valid_range', None)
+        if isinstance(valid_range, np.ndarray):
```

Comment:
What are the other options for what this could be? Would this catch more cases if it was `valid_range is not None`?

Reply:
valid_range is not None is better.